packages/backend/src/spsp/service.ts

```diff
@@ -76,8 +80,9 @@ async function getPay(
   }
 
   try {
+    const account = await deps.accountService.get(invoice.invoiceAccountId)
     const { ilpAddress, sharedSecret } = deps.streamServer.generateCredentials({
-      paymentTag: user.accountId,
+      paymentTag: account.id,
```

Comment:
Why fetch the account instead of just `paymentTag: invoice.invoiceAccountId`?

Reply:
Good catch

Reply:
@sentientwaffle I now remember, you need to get the account as we need to pass in the asset code and scale as well.